code/mathematical-algorithms/armstrong_numbers/armstrong.rb

```diff
@@ -0,0 +1,26 @@
+#created by AnkDos
+#You can Simply use '**' to cube ,ex. 3**3=27 ,so cube(remainder) can be written as (reminder**3) 
+def cube(num)
+    return num*num*num
```

Comment:
no need for the return.

Reply:
Yeah mate, But thats just a habbit.
Thanks!